statsmodels/tsa/arima_process.py

```diff
@@ -108,13 +73,14 @@ def arma_generate_sample(ar, ma, nsample, sigma=1, distrvs=np.random.randn,
     >>> model.params
     array([ 0.79044189, -0.23140636,  0.70072904,  0.40608028])
     """
-    #TODO: unify with ArmaProcess method
-    eta = sigma * distrvs(nsample+burnin)
+    # TODO: unify with ArmaProcess method
+    eta = sigma * distrvs(nsample + burnin)
```

Comment:
There's a PR on deck that handles a few more variants in `np.random` and unifies the function `arma_generate_sample` with the method `ArmaProcess.generate_sample` as per the comment.